packages/istio/charts/1.6.500/templates/istio-install-job.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: istioctl-installer-{{ randAlphaNum 5 | lower }}
```

Comment:
Think it makes more sense to keep name same every time and use: `"helm.sh/hook-delete-policy": before-hook-creation` or delete on success. This way will leave a job for every upgrade. At very least we should set the cleanup TTL.